knotx/conf/openapi.yaml

```diff
@@ -13,8 +13,8 @@ servers:
         description: api domain
       port:
         enum:
-          - '8092'
-        default: '8092'
+          - "8092"
```

Comment:
I'd advise to turn off this kind of linter if you have it on, you shouldn't even touch this file for the scope of this task.

Reply:
This is probably the work of our Prittier which works when committing, I have nothing to do with it: D

Reply:
**if this is prettier issue than we have a problem - it shouldn't touch anything but webapp sources**